src/main/java/mate/academy/spring/boot/service/BookService.java

```diff
@@ -4,20 +4,21 @@
 import mate.academy.spring.boot.dto.BookDto;
 import mate.academy.spring.boot.dto.BookSearchParametersDto;
 import mate.academy.spring.boot.dto.CreateBookRequestDto;
-import mate.academy.spring.boot.model.Book;
+import org.springframework.data.domain.Pageable;
 
 public interface BookService {
-    Book save(Book book);
+    BookDto save(CreateBookRequestDto requestDto);
 
-    List<Book> findAll();
+    List<BookDto> findAll(Pageable pageable);
 
-    List<BookDto> getAll();
+    List<BookDto> getAllBooks();
 
     BookDto getBookById(Long id);
 
     BookDto createBook(CreateBookRequestDto bookDto);
```

Comment:
it looks like the same as BookDto save(CreateBookRequestDto requestDto);. Please, leave one